src/main/java/com/ithaca/BookBudsApplication.java

```diff
@@ -18,6 +21,8 @@ public FilterRegistrationBean jwtFilter() {
 		registrationBean.addUrlPatterns("/users/account");
 		registrationBean.addUrlPatterns("/messages");
```

Comment:
I just looked at the User controller again and no where do I see an endpoint /messages (because the entire controller is /users, the real endpoint would then be /users/messages). That may have been from awhile ago, I am unsure.

As far as I'm aware, everything to do with messages should be in the messages controller, and if not that's something we'll need to think about.